app/src/test/java/com/ikarabulut/GameSettingsTest.java

```diff
@@ -0,0 +1,7 @@
+package com.ikarabulut;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+class GameSettingsTest {
```

Comment:
Empty? :(

Reply:
I wasnt sure if tests would be needed for getters. Its a pretty straightforward method